SalesforceMobileSDK-iOS.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "7.0"
 
   s.source       = { :git => "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git",
-                     :tag => "pod_v#{s.version}",
+                     :tag => "v#{s.version}",
```

Comment:
I changed this back, in anticipation that we'll be re-aligned with the release version tag when this moves forward.